omx/gstomxaacdec.c

```diff
@@ -169,13 +206,19 @@ static gboolean
 gst_omx_aac_dec_is_format_change (GstOMXAudioDec * dec, GstOMXPort * port,
     GstCaps * caps)
 {
-  GstOMXAACDec *self = GST_OMX_AAC_DEC (dec);
+  GstOMXAACDec *self = NULL;
   OMX_AUDIO_PARAM_AACPROFILETYPE aac_param;
   OMX_ERRORTYPE err;
   GstStructure *s;
   gint rate, channels, mpegversion;
   const gchar *stream_format;
 
+  g_return_val_if_fail (dec, NULL);
```

Comment:
Returning `NULL` in a `gboolean` method

Reply:
Ready for review: 0947ed4a695b58965705a115cfebaebbfdd2f0db